orchagent/bufferorch.cpp

```diff
@@ -11,8 +11,6 @@ extern sai_queue_api_t *sai_queue_api;
 extern sai_switch_api_t *sai_switch_api;
 extern sai_buffer_api_t *sai_buffer_api;
 
-extern PortsOrch *gPortsOrch;
-
```

Comment:
In the future, I would recommend you make a commit to remove this global variable, with matching changes like adding parameter to constructor, change until existing code compiles and executes. Then create small commit with only this change. That way review becomes much easier because each commit only tell a very small story.